qml/MonitorItem.qml

```diff
@@ -354,7 +366,7 @@ Component
                 }
 
                 Row {
-                    anchors.right: parent.right
+                    anchors.left: parent.left
```

Comment:
Since users generally read from left to right, positioning the primary action on the right can feel more intuitive. Conversely, secondary or dismissive actions like 'Cancel' or 'Close' are commonly placed on the left.

Please roll back the "anchors" position changes.

Reply:
It looks like the height size of `Cura.ScrollView` needs to be changed, since the UM.Label with description has 2 lines, the actual size of which may be larger than expected.
@Elkin-Vasily , any thoughts?

Reply:
The `anchors` setting for `Column` inside the dialog is missing.
```
Column {
            anchors
            {
                top: parent.top
                left: parent.left
                right: parent.right
                bottom: parent.bottom
            }
```

Reply:
After these changes, the dialog will look like this:
<img width="646" alt="image" src="https://github.com/Jeredian/mks-wifi-plugin/assets/9591166/d618b405-1c53-4014-bd5e-2048afa16264">